lib/helper/Appium.js

```diff
@@ -276,6 +276,26 @@ class Appium extends WebdriverIO {
     return recorder.promise();
   }
 
+  /**
+   * Execute code only in Web mode.
+   *
+   * ```js
+   * I.runInWeb(() => {
+   *    I.waitForElement('#data');
+   *    I.seeInCurrentUrl('/data');
+   * });
+   * ```
+   *
+   * @param {*} fn
+   */
+  runInWeb(fn) {
```

Comment:
will this works with generators? Think, that we should change generators to awync/await, when Node 8 will be LTS